src/Service/EndpointService.php

```diff
@@ -247,9 +238,46 @@ public function getAcceptType(): string
             return 'html';
         case 'application/vnd.openxmlformats-officedocument.wordprocessingml.document':
             return 'docx';
-            break;
+                break;
```

Comment:
Switch is missing a default option (or this function needs a return after the switch?)

Reply:
nvm, checked code. and it has a return at the end